Build Failure on ARM32

Since 8a3d2c258792ba1d0f057780c432f2ccafa66cf1 builds on ARM 32 fail:

```
[ERROR] org.jacoco.core.internal.analysis.LineImplMemoryTest.currentVM -- Time elapsed: 0.011 s <<< FAILURE!
org.junit.ComparisonFailure: 
expected:<...        VALUE
  0   [8                                                 (object header: mark)     N/A
  8   4                                                 (object header: class)    N/A
 12   4   org.jacoco.core.internal.analysis.CounterImpl LineImpl.instructions     N/A
 16   4   org.jacoco.core.internal.analysis.CounterImpl LineImpl.branches         N/A
 20   4                                                 (object alignment gap)    
Instance size: 24 bytes
Space losses: 0 bytes internal + 4 bytes external = 4] bytes total
> but was:<...        VALUE
  0   [4                                                 (object header: mark)     N/A
  4   4                                                 (object header: class)    N/A
  8   4   org.jacoco.core.internal.analysis.CounterImpl LineImpl.instructions     N/A
 12   4   org.jacoco.core.internal.analysis.CounterImpl LineImpl.branches         N/A
Instance size: 16 bytes
Space losses: 0 bytes internal + 0 bytes external = 0] bytes total
```

@Godin I wonder whether this platform should be considered in our code base (for example by making assumptions in the `currentVM` test). Otherwise I can simply exclude the the benchmark module.

--- a/org.jacoco.benchmarks/src/org/jacoco/core/internal/analysis/LineImplMemoryTest.java
+++ b/org.jacoco.benchmarks/src/org/jacoco/core/internal/analysis/LineImplMemoryTest.java
@@ -222,7 +222,7 @@ public class LineImplMemoryTest {
 		}
 		if ("32".equals(System.getProperty("sun.arch.data.model"))) {
 			throw new AssumptionViolatedException(
-					"this test requires 64-bit architecture");
+					"this test does not support 32-bit architecture");
 		}
 		return new CurrentLayouter();
 	}

--- a/org.jacoco.benchmarks/src/org/jacoco/core/internal/analysis/LineImplMemoryTest.java
+++ b/org.jacoco.benchmarks/src/org/jacoco/core/internal/analysis/LineImplMemoryTest.java
@@ -220,6 +220,10 @@ public class LineImplMemoryTest {
 			}
 			throw e;
 		}
+		if ("32".equals(System.getProperty("sun.arch.data.model"))) {
+			throw new AssumptionViolatedException(
+					"this test requires 64-bit architecture");
+		}
 		return new CurrentLayouter();
 	}
 

